va/validation-authority.go

```diff
@@ -36,12 +36,13 @@ var ErrTooManyCNAME = errors.New("too many CNAME/DNAME lookups")
 
 // ValidationAuthorityImpl represents a VA
 type ValidationAuthorityImpl struct {
-	RA           core.RegistrationAuthority
-	log          *blog.AuditLogger
-	DNSResolver  core.DNSResolver
-	IssuerDomain string
-	TestMode     bool
-	UserAgent    string
+	RA            core.RegistrationAuthority
+	log           *blog.AuditLogger
+	DNSResolver   core.DNSResolver
+	IssuerDomain  string
+	TestMode      bool
+	UserAgent     string
+	AddressFilter int
```

Comment:
Same AddrFilter type here.

Why is there only one AddressFilter time per VAImpl? Why are we adding this field and such to this review if only NoAddrFilter is always used? Not quite following, yet.